### DISCUSSION TITLE
Split up the repo in a Hyva-specific and a Luma-specific one?

### DISCUSSION THREAD

**Discussion by peterjaap** [elgentos/magento2-cypress-testing-suite#37] [Ideas]
Instead of having a duplicated folder structure for both themes, maybe its better to maintain two separate repositories? What do you think?

**Comment by phoenix-bjoern** [comment]
There are some things which both projects should share like fixtures, global settings and configuration approach. So actually there should be 3 repos then but I can’t tell if that makes it easier for anyone. 
I would expect that we see support for PWA Studio this year. A more modular approach could help. However, keeping everything in one place encourages maintainers to keep supported themes at the same level.

**Comment by mautz-et-tong** [comment]
When this discussion is about having a config setting `integrationFolder` and `fixturesFolder` to hyva or luma (or pwa or whatever) or having two (or even more) nearly duplicate repositories I am team config. One repo is better for maintenance.
